src/components/bottom_bar_sheet/bottom_bar_sheet.ts

```diff
@@ -185,7 +185,18 @@ export class BottomBarSheet extends Component<Props, SpreadsheetChildEnv> {
   }
 
   private getInputContent(): string | undefined | null {
-    return this.sheetNameRef.el?.textContent;
+    const element = this.sheetNameRef.el;
```

Comment:
I think we can use `contenteditable="plaintext-only"` and it would avoid the issue right ?

```patch
diff --git a/src/components/bottom_bar_sheet/bottom_bar_sheet.xml b/src/components/bottom_bar_sheet/bottom_bar_sheet.xml
index 98de053af..b79f68091 100644
--- a/src/components/bottom_bar_sheet/bottom_bar_sheet.xml
+++ b/src/components/bottom_bar_sheet/bottom_bar_sheet.xml
@@ -19,7 +19,7 @@
           t-on-dblclick="() => this.onDblClick()"
           t-on-focusout="() => this.onFocusOut()"
           t-on-keydown="(ev) => this.onKeyDown(ev)"
-          t-att-contenteditable="state.isEditing.toString()"
+          t-att-contenteditable="state.isEditing ? 'plaintext-only': '0'"
         />
         <span class="o-sheet-icon ms-1" t-on-click.stop="(ev) => this.onIconClick(ev)">
           <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
```

Reply:
Absolutely, @LucasLefevre, Updated the code. Thank you for the input!